clients/fides-js/src/components/Button.tsx

```diff
@@ -5,16 +5,18 @@ interface ButtonProps {
   buttonType: ButtonType;
   label?: string;
   onClick?: () => void;
+  id?: string;
 }
 
 const Button: FunctionComponent<ButtonProps> = ({
   buttonType,
   label,
   onClick,
+  id,
 }) => (
   <button
     type="button"
-    id={`fides-banner-button-${buttonType.valueOf()}`}
+    id={id}
```

Comment:
I think this is a good change since html `id`s should be unique in html, and this resulted in duplicates of the same ID. this change does pose a risk if any one was already relying on this id though! they would be able to swap to use the class (of the same name) instead, but that would be a change

Reply:
Hm, so we do have a customer relying on these ids. Is it possible to use a different approach here, perhaps generating a new class instead?

Reply:
okay, I'll keep the id around and work around this!

Reply:
okay, changed it to an additive classname instead 👍